composer.json

```diff
@@ -52,9 +53,9 @@
         "payum/omnipay-bridge": "~1.0",
         "payum/payum": "~1.0",
         "payum/payum-bundle": "~1.0",
+        "polishsymfonycommunity/symfony-mocker-container": "^1.0",
```

Comment:
This dependency is also missing in `CoreBundle`.

Btw. it should be only required in dev, but it's a separate topic.